test/jasmine/tests/geo_test.js

```diff
@@ -1995,55 +2284,112 @@ describe('Test geo zoom/pan/drag interactions:', function() {
+                ]);
+                return Plotly.relayout(gd, 'geo.showlakes', true);
+            })
+            .then(function() {
+                _assert('after some relayout call that causes a replot', [
+                    [34.3, 43.6], 1.3
+                ], [
+                    // changes are now reflected in 'center'
+                    [247, 260], [19.3, 43.6], 385.5
+                ], [
+                    'geo.showlakes'
+                ]);
+                return dblClick([250, 250]);
+            })
+            .then(function() {
+                _assert('after double click', [
+                    [15, 57.5], 1,
+                ], [
+                    [247, 260], [0, 57.5], 292.2
+                ], 'dblclick');
+            })
+            .catch(failTest)
+            .then(done);
+        });
+
+        it('- fitbounds case', function(done) {
+            fig.layout.geo.fitbounds = 'locations';
+
+            plot(fig).then(function() {
+                _assert('base', [
+                    [undefined, undefined], undefined,
+                ], [
+                    [247, 260], [5.7998, 49.29], 504.8559
+                ], undefined);
+                return drag({path: [[250, 250], [200, 200]], noCover: true});
+            })
+            .then(function() {
+                _assert('after SW-NE drag', [
+                    [29.059, 42.38], 1.727
+                ], [
+                    [197, 210], [5.7988, 49.29], 504.8559
+                ], [
+                    'geo.center.lon', 'geo.center.lon',
+                    'geo.projection.scale', 'geo.fitbounds'
+                ]);
+                return scroll([300, 300], [-200, -200]);
+            })
+            .then(function() {
+                _assert('after scroll', [
+                    [31.027, 40.91], 2.28
+                ], [
+                    [164.09, 181.24], [5.7988, 49.29], 666.16
+                ], [
+                    'geo.center.lon', 'geo.center.lon',
+                    'geo.projection.scale'
+                ]);
+                return Plotly.relayout(gd, 'geo.showlakes', true);
+            })
+            .then(function() {
+                _assert('after some relayout call that causes a replot', [
+                    [31.027, 40.91], 2.28
+                ], [
+                    // changes are now reflected in 'center'
+                    [247, 260], [16.027, 40.91], 666.16
+                ], [
+                    'geo.showlakes'
+                ]);
+                return dblClick([250, 250]);
+            })
+            .then(function() {
+                _assert('after double click', [
+                    [undefined, undefined], undefined,
+                ], [
+                    [247, 260], [5.7998, 49.29], 504.8559
+                ], 'dblclick');
+            })
+            .catch(failTest)
+            .then(done);
+        });
```

Comment:
Thanks very much for adding all these tests!
One of the tests is not passing on my machine:

![geo_test](https://user-images.githubusercontent.com/33888540/70737522-ba2d3b80-1ce0-11ea-8fe7-48d54b31b970.png)

Reply:
Ok thanks for checking. This one passes fine on my machine. I suspect it's related to https://github.com/plotly/plotly.js/issues/2043

Reply:
@archmoj can you try running `npm run test-jasmine -- geo` on your machine now after https://github.com/plotly/plotly.js/pull/4419/commits/2b5ddec0f9a0b75e3efa058a7f69155a0d47c73c ?

Reply:
@etpinard Thanks very much for the fix.
All passed.
Resolving...